convert2rhel/checks.py

```diff
@@ -21,11 +21,13 @@
 import os
 import re
 
+import conver2rhel
```

Comment:
I think a better way of importing this would be like 

```suggestion
from convert2rhel import __version__ as convert2rhel_version
```

This way we don't trigger warnings from `unused module` and don't mess with importing the whole module (if in the future we happen to have more things than usual there)